src/utils.js

```diff
@@ -14,12 +15,35 @@ function getPulumiStack() {
  * Add environment variable to process.env
  *
  * @param {{}} twilioClient Initialized Twilio Client
- * @param {*?} flags Command flags
- * @param {string?} stack Pulumi stack
+ * @param {*?} shouldGetEnvsFromFile Whether to search and load env file for the current stack  
  * @return {{}} Environment key-value pairs
  */
-function getEnvironmentVariables(twilioClient) {
+function getEnvironmentVariables(twilioClient, shouldGetEnvFromFile) {
+
   let envVars = process.env;
+
+  //remove recursion
+  if(shouldGetEnvFromFile) {
+
+    let environment = getPulumiStack();
+
+    if(environment){
+
+      const envFilePath = `${process.cwd()}/.${environment}.env`;
```

Comment:
@kaiquelupo how about we use the naming convention of `.env.<stack>`? I'm not overly opinionated on that, but there are a couple of minor considerations:

* It's the same convention used in the Twilio docs for Serverless (see [here](https://www.twilio.com/docs/labs/serverless-toolkit/guides/multiple-twilio-projects#using-different-environment-variables))
* It makes it easier to find all the `.env` files in the folder since they will be next to each other in sorted alphabetically   

What do you think? @bermudezmt?

Reply:
Liked it! Changing it now